src/python/aqueduct_executor/operators/utils/utils.py

```diff
@@ -159,10 +180,17 @@ def _write_numeric_output(
     output_path: str,
     output_metadata_path: str,
     val: Union[float, int],
+    system_metadata: Dict[str, Any],
 ) -> None:
     """Used for metrics."""
     storage.put(output_path, bytes(str(val), encoding=_DEFAULT_ENCODING))
-    storage.put(output_metadata_path, bytes(json.dumps([]), encoding=_DEFAULT_ENCODING))
+    storage.put(
+        output_metadata_path,
+        bytes(
+            json.dumps({"schema": [], "SystemMetadata": system_metadata}),
```

Comment:
why do we always include the "schema" key, even if non of these output types have schemas?